.github/ISSUE_TEMPLATE/20_feature.md

```diff
@@ -1,6 +1,6 @@
 ---
-name: Feature request
-about: I have an idea for how to improve CFF
+name: Schema/format enhancement
```

Comment:
maybe we prepend the title with "Idea for " to help differentiate "Idea for schema/format change" from "Idea for new tooling"

Reply:
Done in 032ddc066e8c1b5d20d7210d5a80e71df40925cb.